### DISCUSSION TITLE
20.3.0

### DISCUSSION THREAD

**Discussion by davidjgraph** [jgraph/drawio-desktop#1115] [General]
## Releases Notes for 20.3.0
[Windows Installer](https://github.com/jgraph/drawio-desktop/releases/download/v20.3.0/draw.io-20.3.0-windows-installer.exe)
[Windows No Installer](https://github.com/jgraph/drawio-desktop/releases/download/v20.3.0/draw.io-20.3.0-windows-no-installer.exe)
[macOS - Universal](https://github.com/jgraph/drawio-desktop/releases/download/v20.3.0/draw.io-universal-20.3.0.dmg)
Linux - [deb](https://github.com/jgraph/drawio-desktop/releases/download/v20.3.0/drawio-amd64-20.3.0.deb), [snap](https://snapcraft.io/drawio), [AppImage](https://github.com/jgraph/drawio-desktop/releases/download/v20.3.0/drawio-x86_64-20.3.0.AppImage) or [rpm](https://github.com/jgraph/drawio-desktop/releases/download/v20.3.0/drawio-x86_64-20.3.0.rpm)
[Google Chrome OS](https://chrome.google.com/webstore/detail/drawio-desktop/pebppomjfocnoigkeepgbmcifnnlndla/reviews)

Windows intel x32 releases are marked -ia32-

**ChangeLog:**

- Fixes security issue https://huntr.dev/bounties/2d93052f-efc6-4647-9a6d-8b08dc251223/ . It is ranked as high, but the conditions that need to be met are extremely unlikely.
- Updates to [draw.io core 20.3.0](https://github.com/jgraph/drawio/blob/v20.3.0/ChangeLog). All changes from 20.3.0 are added in this build.

<hr /><em>This discussion was created from the release <a href='https://github.com/jgraph/drawio-desktop/releases/tag/v20.3.0'>20.3.0</a>.</em>

**Comment by sire42** [comment]
In the desktop variant, PlantUML is not listed in Arrange->Insert->Advanced Menü, while in online app it is.

**Reply by davidjgraph** [reply]
PlantUML is not supported in the desktop build as it requires communications to a server.

**Reply by stefano5858** [reply]
is this 20.3.0 considered an upgrade in place from a 14.x or an update to the existing?

**Comment by eki78** [comment]
Blank screen when starting, no problem with previous version. Same issue on another computer.

**Comment by zShax** [comment]
Amazing, thank you for making my University life easiser!

**Reply by Easonwenh** [reply]
great！

**Comment by nightfu4y** [comment]
This tool is amazing. Thanks a lot!

**Comment by pequeue** [comment]
居然自带中文，厉害
